### PR TITLE
More details on exceptions.

### PR DESCRIPTION
**Pull request by SpacePossum**
Master version of: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1234

### CODE REVIEW

File: Symfony/CS/Tests/Console/ConfigurationResolverTest.php [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\CS\Tests\Console;
 
+use Guzzle\Common\Exception\UnexpectedValueException;
```
Comment (keradus) [inline Symfony/CS/Tests/Console/ConfigurationResolverTest.php]: ?

Comment (SpacePossum) [inline Symfony/CS/Tests/Console/ConfigurationResolverTest.php]: It is used to resolve this class:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1241/files#diff-84cef3eba62d0b0d72bf5973c56ef3deR463

(this prevent false positives as the code being tested throws an exception with the class name `UnexpectedValueException` but in another namespace)

Comment (keradus) [inline Symfony/CS/Tests/Console/ConfigurationResolverTest.php]: sorry, but I can't see where (and why) `ConfigurationResolver` throws exception from `Guzzle` namespace

Comment (SpacePossum) [inline Symfony/CS/Tests/Console/ConfigurationResolverTest.php]: oh, hold on,....

**Comment by keradus** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**